source/state_representation/src/space/cartesian/CartesianState.cpp

```diff
@@ -237,17 +236,21 @@ CartesianState CartesianState::inverse() const {
   return result;
 }
 
-void CartesianState::clamp_state_variable(double max_value,
-                                          const CartesianStateVariable& state_variable_type,
-                                          double noise_ratio) {
+void CartesianState::clamp_state_variable(
+    double max_norm, const CartesianStateVariable& state_variable_type, double noise_ratio
+) {
+  if (state_variable_type == CartesianStateVariable::ORIENTATION
+      || state_variable_type == CartesianStateVariable::POSE) {
+    throw (NotImplementedException("clamp_state_variable is not implemented for this CartesianStateVariable"));
```

Comment:
There seems to be an unneeded set of parentheses here 
```suggestion
    throw NotImplementedException("clamp_state_variable is not implemented for this CartesianStateVariable");
```